sbt-metals/src/main/scala/MetalsPlugin.scala

```diff
@@ -98,16 +98,16 @@ object MetalsPlugin extends AutoPlugin {
       IO.delete(configDir)
       configDir.mkdirs()
       "semanticdbEnable" ::
-        "*:metalsWriteCompilerConfig" ::
+        "Global / metalsWriteCompilerConfig" ::
```

Comment:
Will this work in sbt 0.13? We should try to keep 0.13 compatibility in MetalsPlugin.scala

Reply:
Doesn't the CI check this? `^compile` should catch it, right?

Reply:
Right, it should be `*/metalsWriteCompilerConfig`. 

@gabro it's just a string, so it compiles 👌 but if you run it on sbt 0.13, it will fail 🌵 We should add scripted tests for the plugin to avoid such mistakes (but in another PR already)